judgments/urls.py

```diff
@@ -26,7 +26,7 @@
         BrowseView.as_view(),
         name="browse",
     ),
-    path("atom.xml", feeds.LatestJudgmentsFeed(), name="feed"),
+    path("atom.xml", feeds.SearchJudgmentsFeed(), name="search-feed"),
     path("<court:court>/atom.xml", feeds.LatestJudgmentsFeed(), name="feed"),
```

Comment:
Could we make these just push a redirect to `/atom.xml` with appropriate params?